ios/App/App/Info.plist

```diff
@@ -17,9 +17,9 @@
     <key>CFBundlePackageType</key>
     <string>APPL</string>
     <key>CFBundleShortVersionString</key>
-    <string>4.7.3</string>
+    <string>4.7.4</string>
     <key>CFBundleVersion</key>
-    <string>221101469</string>
+    <string>221118304</string>
     <key>LSRequiresIPhoneOS</key>
```

Comment:
Skulle ikke dette være 4.7.5?
Sikkert bare glemt å kjøre create-version-file?
Samme for AndroidManifest.xml.